content/kubernetes/re-clusters/multi-namespace.md

```diff
@@ -91,6 +88,81 @@ kubectl apply -f role_binding.yaml
 
 ## Update Redis Enterprise operator ConfigMap
 
+
+There are two methods of updating the operator ConfigMap (`operator-environment-config`) to specify which namespaces to manage.
+
+- Method 1: Configure the operator to watch for a namespace label and add this label to managed namespaces.
+- Method 2: Configure the operator with an explicit list of namespaces to manage.
+
+This ConfigMap can be created manually before deploying the RedisEnterpriseCluster, or will be created automatically when a Redis Enterprise cluster (REC) is deployed.
```

Comment:
```suggestion
You can create this ConfigMap manually before deploying the RedisEnterpriseCluster, or it will be created automatically when a Redis Enterprise cluster (REC) is deployed.
```